File: frontEnd/src/types/dto/stepDTO/IFStepDTO.ts
// if判断操作
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";
import {ConditionalRelationship} from "@/types/enum/condation/ConditionalRelationship";
import {AssertStepDTO} from "@/types/dto/stepDTO/AssertStepDTO";

export class IFStepDTO extends StepBaseDTO {
    stepType: string = 'IF';

    conditionList: AssertStepDTO[] = [];

    // 条件关系
    conditionalRelationship: ConditionalRelationship | string = "AND";
}

File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/IFStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;// if判断操作

import com.mokatest.platform.demos.domain.ui.uiEnum.condation.ConditionalRelationship;
import lombok.Data;

import java.util.List;

@Data
public class IFStepDTO extends StepBaseDTO {
    private String stepType;

    private List<AssertStepDTO> conditionList;


    // 条件关系
    private ConditionalRelationship conditionalRelationship;
}






